src/dynamics/initial_conditions.jl

```diff
@@ -60,42 +60,107 @@ function initialize!(
     has(model, :humid) && initialize!(progn, IC.humid, model)
 end
 
-export StartWithRandomVorticity
+export RandomVorticity
 
 """Start with random vorticity as initial conditions
 $(TYPEDFIELDS)"""
-@kwdef mutable struct StartWithRandomVorticity <: AbstractInitialConditions
-    "Power of the spectral distribution k^power"
+@kwdef mutable struct RandomVorticity <: AbstractInitialConditions
+    "[OPTION] Power of the spectral distribution k^power"
     power::Float64 = -3
 
-    "(approximate) amplitude in [1/s], used as standard deviation of spherical harmonic coefficients"
-    amplitude::Float64 = 1e-5
+    "[OPTION] (approximate) amplitude in [1/s], used as standard deviation of spherical harmonic coefficients"
```

Comment:
this seems the culprit for the docs failing to build!
it's interpreted as a markdown link, e.g., like `[text](link)`! I don't know why, somehow it's parsed without the space...?

Reply:
That's weird! But now with the space it passes, maybe worth flagging in the Documenter repo?